craft_parts/executor/environment.py

```diff
@@ -165,6 +165,8 @@ def _get_global_environment(info: ProjectInfo) -> dict[str, str]:
         "CRAFT_ARCH_TRIPLET_BUILD_FOR": info.arch_triplet_build_for,
         "CRAFT_PARALLEL_BUILD_COUNT": str(info.parallel_build_count),
         "CRAFT_PROJECT_DIR": str(info.project_dir),
+        # Build system configurations for the project.
+        "CARGO_HOME": str(info.dirs.work_dir / "cargo"),
```

Comment:
reminder to remove

Reply:
While I was removing this I came across this: https://github.com/canonical/craft-parts/blob/5418aa74c151570efe537d301e4fcc2186d33ac6/craft_parts/plugins/rust_plugin.py#L223

Re-examining the provenance of both of these leads me to the conclusion that they're two ways to do the same task of making the `rust` plugin work with the `cargo-use` plugin. This one applies it as a blanket change, meaning that other plugins (e.g. the `python` plugin) can use it, while the other applies it only to the `rust` plugin.